src/pages/cli/graphql/data-modeling.mdx

```diff
@@ -449,17 +449,41 @@ When `@default` is applied, non-null assertions using `!` are disregarded. For e
 
 ## Advanced
 
-### Rename generated queries, mutations, and subscriptions
```

Comment:
restore `, mutations, and subscriptions` for consistency with following examples